### PR TITLE
Issue30

### PR DESCRIPTION
**Pull request by xmclark**
Add style guide

### CODE REVIEW

**Comment by xmclark** [comment]
Github is the best. I accidentally set the base to `master` and it let me change it back to `dev`. I was fearing some kind of dreadful git mess was about to unfold, but nope, all good 💃 .

**Comment by technetos** [comment]
Great work!

**Review by technetos — APPROVED** [review]

File: STYLE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<h1 align="center">Rust Style Guide</h1>
+
+*Please write good Rust.*
```
Comment (DannyPeck) [inline STYLE.md:3]: Amen.

**Review by DannyPeck — APPROVED** [review]
Looks great! Looking forward to expanding this in the future.

**Pull request merged**